### ISSUE TITLE
Personal todo lists

### ISSUE THREAD

**Issue by Pont** [alex-LE/yourTinyTodo#34]
At the moment all users share the same todo lists or instance of yourTinyTodo. It would be nice to be able to create personal todo lists that is a completely private instance of yourTinyTodo.
In line with this it would be nice to be able to send a task to another users private instance. Perhaps to a tab for incoming tasks.

**Comment by alex-LE** [comment]
Good idea, i think this can be part of the next release (version 1.0 will be released soon, this can be in version 1.1)

**Comment by jniggemann** [comment]
I also like this idea...

**Comment by alex-LE** [comment]
I've implemented it, have a look

**Comment by jniggemann** [comment]
Cool!

**Comment by jniggemann** [comment]
Seems to work, I enabled multiuser in latest dev, set a list to private as user1, logged ni as user2 and didn't see the private list of user1...

**Comment by electricmessiah** [comment]
The private list is a nice feature, but I don't think it nails down exactly what OP suggested. If we stored the tabs for each user, they could essentially share one instance of the app, but with their own tab lists (archive, private, etc.). We could then link tabs between users so they could collaborate on tasks in those lists.

**Comment by alex-LE** [comment]
Its now possible to move tasks to any (even private) list.

**Comment by idiomsd** [comment]
Any way to make the lists private by default?

**Comment by alex-LE** [comment]
not yet, but it shouldn't be that hard to implement that
